dashboard/src/components/ColorSchemeToggle.tsx

```diff
@@ -0,0 +1,50 @@
+import IconButton, { IconButtonProps } from '@mui/joy/IconButton';
+import { useColorScheme } from '@mui/joy/styles';
+import * as React from 'react';
+
+import DarkModeRoundedIcon from '@mui/icons-material/DarkModeRounded';
+import LightModeIcon from '@mui/icons-material/LightMode';
+
+export default function ColorSchemeToggle(props: IconButtonProps) {
```

Comment:
Other places we are doing export default const and not function? maybe we should keep it as the others?